queries.sql

```diff
@@ -70,3 +70,8 @@ join species s on s.id = species_id
 where v.name = 'Maisy Smith'
 group by specie_name, v.name
 order by count(*) desc limit 1;
+
+/*MileStone 5*/
+CREATE INDEX animal_id_index on visits(animal_id);
+CREATE INDEX vet_id_index on visits(vet_id);
+CREATE INDEX email_id on owners(email);
```

Comment:
- [ ] [OPTIONAL] This is a nice-to-have a suggestion as it won't improve the query that was reported as problematic in the requirements for this project 😃 

As email addresses are not case sensitive, we might need to look for the emails in a case insensitive way. There is an option to reflect that in the index by using:

```
CREATE INDEX ON owners ((lower(email)));
```

Reply:
Thank you so much mate